entity-server-api/src/main/java/org/terracotta/entity/ConcurrencyStrategy.java

```diff
@@ -67,6 +72,15 @@
    * server.  This operation is performed on a per-concurrencyKey basis so this method is used to determine the set of keys
    * which must be synchronized.
    * 
+   * A synchronization operation will be issued for each of the keys returned and run on that key.  This means that the
```

Comment:
When you say "a synchronization operation" - do you mean a call to synchronizeKeyToPassive on the active entity. If so, say that, if not, then say what.